kythe/go/extractors/config/runextractor/backup/backup.go

```diff
@@ -36,10 +36,13 @@ import (
 	"path/filepath"
 )
 
+// A File is a simple holder for a backup of some file that supports Recovery
```

Comment:
I don't think "simple" adds much here. Suggestion:

```
// A File records the locations of an original file and a backup copy of that file.
// The Restore method replaces the contents of the original with the backup,
// overwriting any changes that were made since the backup was created.
```

or something like that.

Reply:
done